### PR TITLE
Add a promotion settings screen

### PR DESCRIPTION
**Pull request by swansontec**
I am tired of hearing that promotions / affiliations "aren't working", so I made a screen to show us what is going on.

Screenshot_20200422-174940_Edge

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [x] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: src/reducers/AccountReducer.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const accountInner: Reducer<AccountState, Action> = combineReducers({
         return { ...referral, promotions }
       }
       case 'ACCOUNT_SWAP_IGNORED': {
-        return { ...state, ignoreAccountSwap: true }
+        return { ...state, ignoreAccountSwap: action.data }
```
Comment (paullinator) [inline src/reducers/AccountReducer.js:40]: This does not look relevant to promotion settings. Was this a bug fix?

Comment (swansontec) [reply src/reducers/AccountReducer.js:40]: No, it's a new feature. I added the ability to cancel the user preferred swap setting and go back to the affiliated one, but then I realized that was stupid & unnecessary & removed the UX but left the action.
